lld/ELF/InputFiles.h

```diff
@@ -9,6 +9,8 @@
 #ifndef LLD_ELF_INPUT_FILES_H
 #define LLD_ELF_INPUT_FILES_H
 
+#include <string>
```

Comment:
Once you remove `StringRef saved_symbol(const std::string &S);` below, you should be able to remove this include also.

Reply:
done.